src/lib/holocene/alert.svelte

```diff
@@ -50,7 +50,7 @@
   {...$$restProps}
 >
   {#if icon}
-    <div>
+    <div class="pt-1">
```

Comment:
Could we just add `items-center` to the `.alert` styles? I hate one-off padding classes for alignment purposes.